docs/topics/mta-cli-args.adoc

```diff
@@ -2,138 +2,165 @@
 //
 // * docs/cli-guide/master.adoc
 
-:_mod-docs-content-type: REFERENCE
+:_content-type: REFERENCE
 [id="cli-args_{context}"]
 = About {ProductShortName} command-line arguments
 
 The following is a detailed description of the available {ProductShortName} command line arguments.
 
 [NOTE]
 ====
-To run the {ProductShortName} command, for example when executing from a script, you must use the following arguments:
+To run the {ProductShortName} command without prompting, for example when executing from a script, you must use the following arguments:
 
+* `--batchMode`
 * `--overwrite`
 * `--input`
 * `--target`
 ====
 
-
 .{ProductShortName} CLI arguments
-[cols="40%,60%a",options="header",]
-|====
-|Argument |Description
-|--analyze-known-libraries | Flag to analyze known software artifacts embedded within your application. By default, {ProductShortName} only analyzes application code.
-
-[NOTE]
+[%collapsible]
 ====
-This option might result in a longer execution time and a large number of migration issues being reported.
-====
-
-|--help |Display the {ProductShortName} help message.
-|--input |A space-delimited list of the path to the file or directory containing one or more applications to be analyzed. This argument is required.
-| --list-sources
-| list rules for available migration sources
-|--list-sources | Flag to list all available source technologies.
-|--list-targets| Flag to list all available target technologies.
-|--output |Specify the path to the directory to output the report information generated by {ProductShortName}.
```

Comment:
All flags in section are still present in CLI.

Reply:
Thanks, I have included them in the table

Please can I ask for clarification on `--list-sources` flag as there are two definitions in the above. Is it correct to say:


| | |
| --- | --- | --- |
|`--list-sources` | |Flag to list rules for available migration sources. |